contrib/postgresql/README.md

```diff
@@ -0,0 +1,123 @@
+Motivation for that second possible postgresql layout:
+------------------------------------------------------
+
+The first layout from Sebastian 'tokkee' Harl is like that:
```

Comment:
Could you make this more general description of that dir contents?  Something like:

> PostreSQL example databases:
> * collectd_insert.sql
> * collectd_insert2.sql
>  
> The first database layout, from Sebastian 'tokkee' Harl, is like this:
> ...